framework/infrastructure/gcp/compute.py

```diff
@@ -203,7 +204,7 @@ def backend_service_patch_backends(
 
         self._patch_resource(
             collection=self.api.backendServices(),
-            body={"backends": backend_list},
+            body={"backends": backend_list, "circuitBreakers": circuit_breaker},
```

Comment:
Let's do this instead: 
```py
        request = {"backends": backend_list}
        if circuit_breaker:
            request["circuitBreakers"] = circuit_breaker

        self._patch_resource(
            collection=self.api.backendServices(),
            body=request,
            backendService=backend_service.name,
        )
```

Just so that we don't make log `circuitBreakers: null` every time we add backends to the backend services (happens in ALL tests)

![image](https://github.com/grpc/psm-interop/assets/672669/9af9149c-9c3f-4a01-b557-22dfcb46ea48)